lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -28,23 +37,13 @@ def get_url(self, subsection):
         request = self.context['request']
         return request.build_absolute_uri(relative_path)
 
-    def get_problem_scores(self, subsection):
-        problem_scores = [
-            {
-                'earned': score.earned,
-                'possible': score.possible,
-            }
-            for score in subsection.problem_scores.values()
-        ]
-        return problem_scores
-
     def get_show_grades(self, subsection):
         return subsection.show_grades(self.context['staff_access'])
 
 
 class ChapterSerializer(serializers.Serializer):
```

Comment:
Should this maybe be renamed? Even when you assign it below, it's assigned as `courseware_point_summary`. I don't know what the data looks like, so maybe this makes sense in context, just want to check.